### PR TITLE
Update free-courses-id.md

### PR DESCRIPTION
**Pull request by Zura79**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s)

## For resources
Add Kursus Javascript 8 jam untuk pemula,15 Projek Javascript dengan vanilla javascript and Pemrograman berbasis objek Javascript
### Why is this valuable (or not)?
This course really help you which is still beginner and unfamiliar with javascript language.
I ensure after you watch this course and do some exercise you will get mid-end javascript skills

### How do we know it's really free?
It's public video

### For book lists, is it a book? For course lists, is it a course? etc.
It's a video course

### Checklist:
- [🗸] Not a duplicate
- [🗸] Included author(s) if appropriate
- [🗸] Lists are in alphabetical order
- [🗸] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

**Comment by SethiPandi** [comment]
Heads up, you can just replace the space with an x in the Pull Request template to make it a checkbox!

There should be two line breaks above new sections, please see the build errors on Travis or read the contributions guidelines for more information there.

Coould you also change this so it's it alphabetical order?

**Comment by Zura79** [comment]
> in the Pull Req

**Comment by Zura79** [comment]
> Heads up, you can just replace the space with an x in the Pull Request template to make it a checkbox!
> 
> There should be two line breaks above new sections, please see the build errors on Travis or read the contributions guidelines for more information there.
> 
> Coould you also change this so it's it alphabetical order?

what do you mean about replace space with x ?

**Comment by SethiPandi** [comment]
> > Heads up, you can just replace the space with an x in the Pull Request template to make it a checkbox!
> > There should be two line breaks above new sections, please see the build errors on Travis or read the contributions guidelines for more information there.
> > Coould you also change this so it's it alphabetical order?
> 
> what do you mean about replace space with x ?

* [ ] Unchecked `* [ ]`
* [x] Checked `* [x]`
* [🗸] What you did `* [🗸]`

**Comment by Zura79** [comment]
> ## Hacktoberfest notes:
> * due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
> * please read our [contributing guidelines](/CONTRIBUTING.md)
> * be sure to check the output of Travis-CI for linter errors
> * if this is your first open source contribution, make sure it's not your last!
> 
> ## What does this PR do?
> Add Resource(s)
> 
> ## For resources
> Add Kursus Javascript 8 jam untuk pemula,15 Projek Javascript dengan vanilla javascript and Pemrograman berbasis objek Javascript
> 
> ### Why is this valuable (or not)?
> This course really help you which is still beginner and unfamiliar with javascript language.
> I ensure after you watch this course and do some exercise you will get mid-end javascript skills
> 
> ### How do we know it's really free?
> It's public video
> 
> ### For book lists, is it a book? For course lists, is it a course? etc.
> It's a video course
> 
> ### Checklist:
> * [ x ] Not a duplicate
> * [ x ] Included author(s) if appropriate
> * [ x ] Lists are in alphabetical order
> * [  ] Needed indications added (PDF, access notes, under construction)

**Comment by eshellman** [comment]
For youtube, we prefer "playlist" links and the name of the producer/creator. Please no shortened links.

**Comment by eshellman** [comment]
no shortened links, please

**Comment by Zura79** [comment]
> no shortened links, please

where is the shortened link part ?? i don't add any shortened link

**Comment by Zura79** [comment]
> youtu.be links

can you tell me where it is?? i can't find it

**Comment by eshellman** [comment]
in the react section

File: free-courses-en.md [hunk 1]
```diff
@@ -299,10 +299,13 @@
 
 #### React
 
+* [Build a Discord Clone with REACT JS for Beginners](https://youtu.be/zc1loX80TX8) - Clever Programmer
```
Comment (eshellman) [inline free-courses-en.md:302]: shortened link

File: free-courses-en.md [hunk 2]
```diff
@@ -299,10 +299,13 @@
...
+* [Build a YouTube Clone with REACT JS for Beginners](https://youtu.be/NT299zIk2JY) - Clever Programmer
```
Comment (eshellman) [inline free-courses-en.md:303]: no shortened links!

**Pull request closed without merging**